pkg/crc/preflight/preflight_checks_network_linux.go

```diff
@@ -72,22 +72,26 @@ server=/crc.testing/192.168.130.11
 dns=dnsmasq
 `
 
-	crcNetworkManagerDispatcherPath   = filepath.Join(crcNetworkManagerRootPath, "dispatcher.d", "pre-up.d", "99-crc.sh")
+	crcNetworkManagerDispatcherPath   = filepath.Join(crcNetworkManagerRootPath, "dispatcher.d", "99-crc.sh")
```

Comment:
Are we going to remove the old dispatcher file, in case user already using old version and if it going to have any kind of conflict with existing one.

Reply:
Ah, good point, totally forgot this. Should be fixed now, thanks!